WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverCoordinator.swift

```diff
@@ -0,0 +1,104 @@
+import UIKit
+
+final class CompliancePopoverCoordinator {
+    fileprivate enum Constants {
```

Comment:
`fileprivate` is a bit of a code smell.

Given the fact that you really want to access this from `UserDefaults`, WDYT about moving it to the the extension?